keyboards/cannonkeys/ortho48v2/info.json

```
@@ -0,0 +1,96 @@
+{
+    "keyboard_name": "Ortho48 v2",
+    "maintainer": "awkannan",
+    "manufacturer": "CannonKeys",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "usb": {
+        "vid": "0xCA04",
+        "pid": "0x0018",
+        "device_version": "0.0.1"
+    },
+    "url": "https://cannonkeys.com",
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": ["GP7", "GP6", "GP5", "GP4", "GP3", "GP2", "GP1", "GP0", "GP13", "GP24", "GP23", "GP22"],
+        "rows": ["GP18", "GP19", "GP25", "GP26"]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "backlight": true,
+        "encoder": true,
+        "encoder_map": true
```

`encoder_map` should never be enabled at the keyboard level, only keymap.
```suggestion
        "encoder": true
```